cli/cmd/net.go

```diff
@@ -44,15 +43,20 @@ func get(url string, display bool) (map[string]interface{}, error) {
 		panic(err)
 	}
 
-	if err := json.Unmarshal(data, &dat); err != nil {
-		panic(err)
-	}
-
-	if display {
-		s, _ := prettyjson.Marshal(dat)
-		fmt.Println(string(s))
-	}
-	return dat, nil
+  if resp.StatusCode == 200 {
+		var dat map[string]interface{}
+		if err := json.Unmarshal(data, &dat); err != nil {
+			panic(err)
+		}
+
+		if display {
+			s, _ := prettyjson.Marshal(dat)
+			fmt.Println(string(s))
+		}
+		return dat, nil
+	} 
+	fmt.Println(string(data))
+	return nil, nil
```

Comment:
is it normal to return `nil, nil`?
by the way, in this function you never return an error, maybe you don't want to. so change the signature of the function

Reply:
I've removed the err. You are right